free-science-books-de.md

```diff
@@ -1,5 +1,18 @@
 # Fachrichtungen
 
+## Biologie
+
+* [Udo Dammer: Zimmerpflanzen (1908)](https://www.gutenberg.org/cache/epub/22823/pg22823-images.html)
```

Comment:
```suggestion
* [Zimmerpflanzen (1908)](https://www.gutenberg.org/cache/epub/22823/pg22823-images.html) - Udo Dammer
```
Sorry I did not have time to do a good review. 
the other files in the rep use the same syntax as free-programming-books, with author after the link.
Also, the linter for the repo has not been setup, so you weren't alerted to the fact that the lists should be in alphabetical order. Thanks for understanding!